govcd/openapi.go

```diff
@@ -494,7 +494,7 @@ func (client *Client) OpenApiDeleteItem(apiVersion string, urlRef *url.URL, para
 // OpenApiPostItemAsync, OpenApiPostItem, OpenApiPutItemSync, OpenApiPutItemAsync, OpenApiPutItem
 func (client *Client) openApiPerformPostPut(httpMethod string, apiVersion string, urlRef *url.URL, params url.Values, payload interface{}, additionalHeader map[string]string) (*http.Response, error) {
 	// Marshal payload if we have one
-	var body *bytes.Buffer
+	body := new(bytes.Buffer)
```

Comment:
This change is required so that `==nil` evaluation works well when an empty body is being sent.
var body *bytes.buffer (!= nil)
new(bytes.Buffer) == nil